src/snowflake/cli/api/plugins/command/__init__.py

```diff
@@ -28,6 +28,8 @@
 
 plugin_hook_spec = pluggy.HookspecMarker(SNOWCLI_COMMAND_PLUGIN_NAMESPACE)
 plugin_hook_impl = pluggy.HookimplMarker(SNOWCLI_COMMAND_PLUGIN_NAMESPACE)
+lsp_plugin_hook_spec = pluggy.HookspecMarker(SNOWCLI_COMMAND_PLUGIN_NAMESPACE)
+lsp_plugin_hook_impl = pluggy.HookimplMarker(SNOWCLI_COMMAND_PLUGIN_NAMESPACE)
```

Comment:
Should these use the same namespace?

Reply:
Yeah, they can use the same namespace and just use different hook markers. It seemed simpler to not need to configure another plugin manager for another namespace, since SnowCLI isn't using pluggy for discovery on builtins.

Reply:
You also have some other Markers in another file, are they dupes?

Reply:
Yes, dupes. Let me figure out which to keep where.